### ISSUE TITLE
Tooltip: Usage tab

### ISSUE THREAD

**Issue by sstrubberg** [carbon-design-system/carbon-website#3412]
**Ref issue:** https://github.com/carbon-design-system/carbon-website/issues/3332
**Ref Usage tab PDF comments:** https://ibm.ent.box.com/file/1000660824957?s=p07zoqxau8xw7ounitiwr9ha2i5tyqsj

-----

### Overall high level comments
- [x] Remove all image examples and references using the info icon.
- [x] Remove most references of "popover" to clean up content.

#### Description
- [x] Update wording

### Overview
- [x] Update image to use the image that is currently in the "When to use" section. And rewrite paragraph as follows: 
> Although Carbon tooltips previously allowed for interactive elements such as buttons and links, do not include these elements in tooltips; use toggle tips instead.
- [x] **When to use:** Reword the first line "Describe icon buttons", to be more along the lines of "Expose names of controls that lack labels." Remove image.
- [x] **When not to use:** Change first don't image to have focus on the input field, to which it then reveals the helper text in a tooltip.
- [x] **Tooltips versus Toggletips:** Add a image to visually show the difference between the two.
- [x] **Variants:** Remove "Standard tooltip". Remove from content and image example.

### Formatting
- [x] **Standard tooltip anatomy:** Update section to clearly state content in this tooltip can not be critical to user workflows/completing tasks. It is simply additional/option information for the user to view.
- [x] **Definition tooltip anatomy:** Specify the underline is dotted and that it can be activated on hover or focus.
**Placement:**
  - [x] To be placed in view. Omit the word "properly".
  - [x] The image caption should say "Tooltip containers can be automatically or manually positioned to the right, left, bottom, or top."
  - [x] The paragraph under the image is confusing and talks about Toggletips and not Tooltips, revise this section more.
  - [x] The image under this section should have a period in the tooltip.

### Content
- [x] **Text:** Revise section with PDF comments.

### Universal behaviors
- [x] **Keyboard:** Revise section with PDF comments.
- [x] **Screen readers:** Remove section.

### Standard tooltip
- [x] Add more guidance of when and why to use this.

### Icon button and Definition tooltip
- [x]  Revise section with PDF comments.
